File: public/examples/integrations/toolkits/search/walmart/get_walmart_product_details_example_call_tool.py
from arcadepy import Arcade
from arcade_search.enums import WalmartSortBy

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

TOOL_NAME = "Search.GetWalmartProductDetails"

tool_input = {
    "item_id": "1234567890",
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
)
print(response)

File: public/examples/integrations/mcp-servers/search/walmart/get_walmart_product_details_example_call_tool.js
import { Arcade } from "@arcadeai/arcadejs";

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const TOOL_NAME = "Walmart.GetProductDetails";

const toolInput = {
	item_id: "1234567890",
};

const response = await client.tools.execute({
	tool_name: TOOL_NAME,
	input: toolInput,
});

console.log(response);
